plugins/inventory/cobbler.py

```diff
@@ -94,16 +94,29 @@
         description: Prefix to apply to cobbler groups.
         default: cobbler_
       want_facts:
-        description: Toggle, if V(true) the plugin will retrieve host facts from the server.
+        description: Toggle, if V(true) the plugin will retrieve all host facts from the server.
         type: boolean
         default: true
       want_ip_addresses:
         description:
-          - Toggle, if V(true) the plugin will add a C(cobbler_ipv4_addresses) and C(cobbleer_ipv6_addresses) dictionary to the defined O(group) mapping
+          - Toggle, if V(true) the plugin will add a C(cobbler_ipv4_addresses) and C(cobbler_ipv6_addresses) dictionary to the defined O(group) mapping
             interface DNS names to IP addresses.
         type: boolean
         default: true
         version_added: 7.1.0
+      facts_level:
+        description:
+          - "normal: Gather only system-level variables"
+          - "as_rendered: Gather all variables as rolled up by Cobbler"
```

Comment:
Please use full sentences in descriptions.
```suggestion
          - "Set to V(normal) to gather only system-level variables."
          - "Set to V(as_rendered) to gather all variables as rolled up by Cobbler."
```